executor/executor.go

```
@@ -1083,6 +1083,9 @@ func (e *MaxOneRowExec) Next(ctx context.Context, chk *chunk.Chunk) error {
 
 	childChunk := e.children[0].newFirstChunk()
 	err = e.children[0].Next(ctx, childChunk)
+	if err != nil {
+		return errors.Trace(err)
+	}
```

So that was an ignored error. A good demonstration of using the error check.